k8s.yaml

```diff
@@ -0,0 +1,200 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: git2grid
+  namespace: git2grid
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  minReadySeconds: 5
+  template:
+    metadata:
+      labels:
+        app: git2grid
+        role: git2grid
+    spec:
+      containers:
+        - name: git2grid
+          image: billytrend/git2grid:2
+          ports:
+            - containerPort: 3000
+          env:
+            - name: GO_ENV
+              value: "production"
+            - name: eventgridEndpoint
+              valueFrom:
+                secretKeyRef:
+                  name: git2grid-secrets
+                  key: eventgrid-endpoint
+            - name: webhookSecret
+              valueFrom:
+                secretKeyRef:
+                  name: git2grid-secrets
+                  key: webhook-secret
+            - name: eventgridSecret
+              valueFrom:
+                secretKeyRef:
+                  name: git2grid-secrets
+                  key: eventgrid-secret
+---
+apiVersion: extensions/v1beta1
```

Comment:
The cluster already has an nginx ingress controller, no need for all this ceremony to get traffic to our app. I -highly- recommend you create a Helm chart with `helm create`, the default template includes integration with the ingress controller and an easy way to enable HTTPS support through Let's Encrypt.

Regardless, all we need for this app is:
1. The deployment
2. A service pointing to it
3. An ingress pointing to the service

All the other resources in here can be removed.